openmetadata-ui/src/main/resources/ui/src/utils/CommonUtils.tsx

```diff
@@ -786,9 +786,7 @@ export const showPagination = (paging: Paging) => {
 };
 
 export const formatNumberWithComma = (number: number) => {
-  return new Intl.NumberFormat('en-US', { maximumSignificantDigits: 3 }).format(
-    number
-  );
+  return new Intl.NumberFormat('en-US').format(number);
```

Comment:
I feel this will break at other places. Can you verify that?
Does it format numbers with commas properly?

Reply:
Yes its format's properly
<img width="1512" alt="image" src="https://user-images.githubusercontent.com/71748675/200551796-2d54ed97-2d36-41f7-9c25-bd8876d8fc82.png">